src/makeRobotAccountant.js

```diff
@@ -7,6 +7,13 @@
 
 function makeRobotAccountant() {
   // write code here
+  let count = 0;
+
+  return (a) => {
+    count++;
+
+    return (b) => (!(count % 2) && count > 3) ? 'Bzzz... Error!' : a + b;
```

Comment:
i'm sure mentor will ask you put condition in const to improve readability

Reply:
Yep, will change this now